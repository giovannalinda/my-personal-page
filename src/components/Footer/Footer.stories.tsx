import { Meta, Story } from '@storybook/react'
import { Footer } from './Footer'

export default {
  title: 'Components/Footer',
  component: Footer,
} as Meta

export const Default: Story = () => <Footer />
