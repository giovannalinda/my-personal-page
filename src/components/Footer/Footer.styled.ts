import styled from 'styled-components'
import { theme } from '~/styles'

export const Footer = styled.footer`
  display: flex;
  align-items: center;
  justify-content: center;
  padding-bottom: ${theme.space[5]};
  border-top: 1px solid ${theme.colors.gray800};
`

export const Paragraph = styled.p`
  text-align: center;
  font-size: ${theme.font.sizes.small};
  padding-top: ${theme.space[5]};
`
