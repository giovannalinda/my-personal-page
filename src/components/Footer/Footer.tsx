import * as S from './Footer.styled'

export function Footer() {
  return (
    <S.Footer>
      <S.Paragraph>Giovanna Souza Â© Todos os direitos reservados</S.Paragraph>
    </S.Footer>
  )
}
