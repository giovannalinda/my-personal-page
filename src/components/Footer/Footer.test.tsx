import { render } from '~/__helpers__/app-tests'
import { Footer } from './Footer'

describe('Footer', () => {
  it('matches snapshot', () => {
    const { container } = render(<Footer />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
