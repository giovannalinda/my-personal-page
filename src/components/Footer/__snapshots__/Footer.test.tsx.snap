// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<footer
  class="sc-bczRLJ hWHKJc"
>
  <p
    class="sc-gsnTZi bNwlIG"
  >
    Giovanna Souza Â© Todos os direitos reservados
  </p>
</footer>
`;
